tests/PHPStan/Analyser/NodeScopeResolverTest.php

```diff
@@ -427,7 +429,9 @@ public function dataFileAsserts(): iterable
 
 		if (self::$useStaticReflectionProvider || PHP_VERSION_ID >= 70400) {
 			yield from $this->gatherAssertTypes(__DIR__ . '/data/arrow-function-types.php');
-			yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-4902.php');
+			if (PHP_VERSION_ID < 80000) {
+				yield from $this->gatherAssertTypes(__DIR__ . '/data/bug-4902.php');
```

Comment:
Why it's no longer tested on PHP 8+? If there are different asserts, please create `bug-4902-php8.php` for that.